File: js/stockPicker.js
exports.picker = function(prices) {
    const dumpArray = [];
    let count = 0
    for (let i = 0; i < prices.length; i++) {
        for (let j = i+1; j < prices.length; j++){
            if (prices[i] >= prices[j]) {
                continue
            } else {
                if (prices[j] - prices[i] > count) {
                    count = prices[j] - prices[i]
                    dumpArray.pop()
                    dumpArray.push([i, j])
                }
            }
        }
    }
    return dumpArray.flat()
}

File: python/stock_picker.py
def picker(prices):
    ans_list = []
    count = 0
    for i, num in enumerate(prices):
        for x in range(1, len(prices), 1):
            if num > prices[x]:
                pass
            elif num < prices[x] and i < x:
                if prices[x] - num > count:
                    count = prices[x] - num
                    if ans_list:
                        ans_list.append([i,x])
                        ans_list.pop(0)
                    else:
                        ans_list.append([i,x])
    return ans_list[0]
